.github/workflows/publishpackage.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish Python Package
+
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

Comment:
Worth pinning to something more static?

Reply:
It probably doesn't matter too much, since everything is running in docker (and all of github's examples use this). But I have pinned it to 18.04.